frontend/models/product.ts

```diff
@@ -90,10 +100,46 @@ function getVariants(shopifyProduct: NonNullable<FindProductQuery['product']>) {
          specifications: variant.specifications?.value ?? null,
          warranty: variant.warranty?.value ?? null,
          crossSellVariants: getCrossSellVariants(variant),
+         disableWhenOOS: variant.disableWhenOOS?.value === 'true',
```

Comment:
Oh my are these booleans serialized as strings? I thought we added a metafield definition that made `disable_when_oos` a boolean, and I would expect the storefront API to send you a boolean. @sterlinghirsh does that sound right?

Reply:
Metafield values on storefront api seems to be always string: https://shopify.dev/api/storefront/2022-10/objects/metafield